docs/zz_generated.seed.yaml

```diff
@@ -343,6 +343,8 @@ spec:
       weekly:
         # Interval defines the number of days consulted in the metering report.
         interval: 7
+        # Retention defines the number of days after which reports are queued for removal. The default value of "0" keeps reports forever.
```

Comment:
Not sure if this makes sense, but can we add a hint that the retention uses the storage backend's lifecycle mechanism and so the backend needs to support that? I'm not sure there are S3 implementations out there that don't, but .. maybe?

Reply:
Will add some more description to make it more explicit about lifecycle management mechanism.